rst2pdf/createpdf.py

```diff
@@ -192,7 +192,16 @@ def __init__(self, stylesheets=[],
         self.fit_mode = fit_mode
         self.background_fit_mode = background_fit_mode
         self.to_unlink = []
-        self.smarty = smarty
+
+        # See https://pythonhosted.org/smartypants/reference.html#smartypants-module
+        self.smarty = 0
```

Comment:
Can we rename this property to `smartypants_attributes` so that we know it's not the same as the `smarty` variable which holds the command line value of (0, 1 ,2 or 3)?